### ISSUE TITLE
iOS Physical Device not Appearing

### ISSUE THREAD

**Issue by haIIux** [ProxymanApp/Proxyman#1811]
I'm attempting to monitor traffic from my physical device (iPhone 13, iOS 17.0). I've followed the instructions for downloading, installing and enabling the Trust Certificate for Proxyman. I've also added the proxy to my network on my iOS device with the ip address shown in the MacOS app help display. 

I am unable to see the device listed on my Proxyman application on my Mac and I'm unsure why, or how to proceed with fixing it.

**Comment by NghiaTranUIT** [comment]
>  followed the  instructions for downloading, installing and enabling the Trust Certificate for Proxyman

May I ask: You mean it's on your iOS device right? Can you confirm you can you can see Proxyman certificate on Setting app -> General -> Trust Setting -> Proxyman?

**Comment by haIIux** [comment]
Yes. Please see the photo below as I believe that's what you're asking. 

![image](https://github.com/ProxymanApp/Proxyman/assets/47156971/3354a9a2-349c-4d09-b693-86723cf33d4a)

**Comment by NghiaTranUIT** [comment]
I believe that it's an iOS 17 bug. Setting the HTTP Proxy on Wifi doesn't work.

If you don't mind, can you try this:
- You can use invalid IP and Port, for example 192.169.1.199, port 9999 -> If you can access the Internet without any problems -> It indicates it's Apple bug ✅ 

<img width="804" alt="Screenshot 2023-10-17 at 08 49 05" src="https://github.com/ProxymanApp/Proxyman/assets/5878421/2b78f910-3a73-4989-8c60-d18c333d0157">

**Comment by NghiaTranUIT** [comment]
This Apple issue is not 100% reproducible. It works fine with my iOS 17 devices, but not on my iPad. Tried to forget the Wifi and reconnect again might fix it.

To workaround, you can use [Atlantis framework](https://github.com/ProxymanApp/atlantis), which is developed by Proxyman. It captures all data and send to Proxyman macOS to preview.
